example/go-chi/main.go

```diff
@@ -4,9 +4,9 @@ import (
 	"log"
 	"net/http"
 
+	httpSwagger "github.com/Dysar/http-swagger"
```

Comment:
We wont accept a pr based on 3'rd party packages like github.com/Dysar/http-swagger